README.md

```diff
@@ -27,6 +27,7 @@ The node-instance-manager provides a shell with the following commands:
 - `start-excluded [app]` - Starts apps dependencies but not app itself
 - `start-only [app]` - Starts app only (without dependencies)
 - `start-all` - Starts all apps from config
+- `start-profil [profil]` - Starts all APPS which are inside a configured profil
```

Comment:
Unfortunately the proper english term for what you want is `profile`. 😀  Can we change the wording everywhere from `profil` to `profile`?